packages/config/test/global-bin-dir.test.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="../../../typings/index.d.ts"/>
+import path from 'path'
+import { homedir } from 'os'
+import getConfig from '@pnpm/config'
+
+jest.mock('@zkochan/npm-conf/lib/conf', () => {
+  const originalModule = jest.requireActual('@zkochan/npm-conf/lib/conf')
+  class MockedConf extends originalModule {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    constructor (base: any, types: any) {
+      super(base, types)
+      this.prefix = this.globalPrefix = path.join(__dirname, 'global-bin-dir')
+      this.localPrefix = __dirname
+    }
+
+    loadPrefix () {}
+  }
+  return MockedConf
+})
+
+test('respects global-bin-dir in npmrc', async () => {
+  const { config } = await getConfig({
+    cliOptions: {
+      global: true,
+    },
+    packageManager: {
+      name: 'pnpm',
+      version: '1.0.0',
+    },
+  })
+  expect(config.bin).toBe(path.join(homedir(), 'Library', 'pnpm'))
```

Comment:
I assume this test will fail on Windows.

Reply:
Observe whether this is fixed.